internal/admin/server_test.go

```diff
@@ -9,7 +9,22 @@ import (
 )
 
 var expectedMetrics = []string{
```

Comment:
looks like on go 1.21 we got more than these: https://github.com/reddit/baseplate.go/actions/runs/6173805147/job/16756970260?pr=629

maybe instead of using diff for this test (equal), we only check that all the `expectedMetrics` are reported instead (minimal)?

Reply:
The other thing we could do is create our own custom filter and only include the metrics we want.

```
  RedditMetricsGC = collectors.GoRuntimeMetricsRule{regexp.MustCompile(`^/gc/.*`)}
```

Reply:
the new ones added in go 1.21 are also `go_gc_.*`, it just report more gc metrics in go 1.21 so that likely won't work,

Reply:
I meant that we could customize this regexp, not use the default.